app/models/game.rb

```diff
@@ -43,7 +43,7 @@ def populate_board!
   end
 
   def occupied?(row, col)
-    piece.where(row_position: row, col_position: col).present?
+    Piece.where(row_position: row, col_position: col).present?
   end
```

Comment:
the correct fix would be pieces.  Can you see why Piece will cause erroneous behavior?